MojioSDK/mojio-sdk-rest/src/main/java/io/moj/mobile/android/sdk/rest/util/Trace.java

```diff
@@ -0,0 +1,112 @@
+package io.moj.mobile.android.sdk.rest.util;
+
+import android.util.Log;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+/**
+ * Utility log class that will strip out configured log calls.
+ */
+@SuppressWarnings("unused")
+public class Trace {
```

Comment:
I like having this in the SDK, though we might want to consider using something universal like SLF4J at some point so clients can configure the SDKs logging as well (http://www.slf4j.org/android/). Jake Wharton also has a library called Timber but I'm not sure what to think of it.

Not anything actionable for this PR, I just figure that if we have time a good logging library would be a good thing to look into